src/Sulu/Bundle/CoreBundle/Composer/ScriptHandler.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the MassiveSearchBundle
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CoreBundle\Composer;
+
+/**
+ * A handler for general tasks executed with composer scripts.
+ */
+class ScriptHandler
+{
+    const GIT_IGNORE_FILE = '.gitignore';
+
+    /**
+     * Removes the composer.lock file from .gitignore, because we don't want the composer.lock to be included in our
+     * repositories, but they should be included when developing specific project.
+     */
+    public static function removeComposerLockFromGitIgnore()
+    {
+        if (!file_exists(static::GIT_IGNORE_FILE)) {
+            return;
```

Comment:
@chirimoya WDYT? Just do nothing if the file doesn't exist should be fine, right?